### PR TITLE
Fixed the bug #3859

### LINKED ISSUE #3859: 🐛 Bug:`_APP_SMS_PROVIDER` not working.
### 💭 Description

What is the format for `_APP_SMS_PROVIDER`?

I was using `_APP_PHONE_PROVIDER=phone://<Account SID>:<Auth Token>@twilio` on `v0.15.3`. It was working fine.
I updated to `v1.0.1` and using `_APP_SMS_PROVIDER=sms://<Account SID>:<Auth Token>@twilio` as shown in the new docs. Appwrite is returning `Instance of 'Token'` but the request is not being sent to the `twilio`.

### 👀 Have you spent some time to check if this issue has been raised before?

- [x] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by Faizan-Pasha**
### FIXED #3859 

Updated the reference of  [class](https://github.com/appwrite/appwrite/issues/3859#issuecomment-1249833928).
Tested it and it works fine✌️.

### CODE REVIEW

**Review by eldadfux — APPROVED** [review]

File: app/workers/messaging.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 class MessagingV1 extends Worker
 {
-    protected ?SMS $sms = null;
+    protected ?Adapter $sms = null;
```
Comment (stnguyen90) [inline app/workers/messaging.php:22]: We would love to have a test case for this. We were thinking of updating the Mock adapter to send the secret to the request catcher and then have the e2e test verify. Is that something you'd be able to add?

Comment (DynamiteFP) [reply app/workers/messaging.php:22]: Would love to do it. But I don't have any background with PHP. Yesterday itself I looked over PHP's basic syntax so I cloud contribute to `APPWRITE`. I need some time to fully understand the language.

Comment (DynamiteFP) [reply app/workers/messaging.php:22]: @stnguyen90 
What do you mean by test case?

Comment (Meldiron) [reply app/workers/messaging.php:22]: Hey there 👋 I have created tests in a separate PR, it was a pretty complex stuff. I think we can merge your PR, and later merge my PR that improves tests logic.

**Review by Meldiron — APPROVED** [review]
Looks good to me 😇

**Review by stnguyen90 — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]
Awesome work! Thank you for the contribution! 🚀

**Pull request merged**